src/main/java/com/palantir/common/streams/MoreStreams.java

```diff
@@ -30,6 +38,48 @@
     private static final boolean NOT_PARALLEL = false;
 
     /**
+     * Given a {@link Stream<CompletableFuture<U>>}, this function will return a {@link Stream<U>}
```

Comment:
I don't think link works like this. Use `@code` instead.

Reply:
done